releasenotes-builder/src/main/java/com/github/checkstyle/publishers/XdocPublisher.java

```diff
@@ -57,56 +45,16 @@ public class XdocPublisher {
     private final String localRepoPath;
     /** Path to a local git repository. */
     private final String postFilename;
-    /** Release number. */
-    private final String releaseNumber;
-    /** Whether to do push. */
-    private final boolean doPush;
-    /** Auth token for Github. */
-    private final String authToken;
 
     /**
      * Default constructor.
      *
      * @param postFilename the name of the file to get post xml from.
      * @param localRepoPath path to a local git repository.
-     * @param releaseNumber release number.
-     * @param doPush whether to do push.
-     * @param authToken auth token for Github.
      */
-    public XdocPublisher(String postFilename, String localRepoPath, String releaseNumber,
-            boolean doPush, String authToken) {
+    public XdocPublisher(String postFilename, String localRepoPath) {
         this.postFilename = postFilename;
         this.localRepoPath = localRepoPath;
-        this.releaseNumber = releaseNumber;
-        this.doPush = doPush;
-        this.authToken = authToken;
-    }
-
-    /**
-     * Publish release notes.
-     *
-     * @throws IOException if problem with access to files appears.
-     * @throws GitAPIException for problems with jgit.
-     */
-    public void publish() throws IOException, GitAPIException {
-        changeLocalRepoXdoc();
```

Comment:
private method `changeLocalRepoXdoc` remains but is not being used anywhere.

Reply:
Right, Did Changes.